src/pages/moderator/buzz-withdrawal-requests.tsx

```diff
@@ -393,10 +497,27 @@ export default function ModeratorBuzzWithdrawalRequests() {
                       </Badge>
                     </td>
                     <td align="right">
-                      <Group noWrap>
-                        {buttons.map((btn) => btn)}
-                        <BuzzWithdrawalRequestHistory history={request.history} />
-                      </Group>
+                      {selectionEnabled ? (
+                        <Checkbox
+                          checked={isSelected}
+                          onChange={(event) => {
+                            console.log('clickeo si');
+                            setSelection((curr) => ({
+                              ...curr,
+                              values: !isSelected
+                                ? [...curr.values, request.id]
+                                : curr.values.filter((v) => v !== request.id),
+                            }));
+                          }}
+                          disabled={request.status !== BuzzWithdrawalRequestStatus.Requested}
+                          radius="lg"
+                        />
+                      ) : (
+                        <Group noWrap>
+                          {buttons.map((btn) => btn)}
```

Comment:
I guess we can simply do
```suggestion
                          {buttons}
```